build/build.sh

```diff
@@ -37,7 +37,6 @@ if [ "${go_version:0:3}" = "1.4" ]; then
 fi
 
 ldflags="
-  -extldflags '-static'
```

Comment:
@tianshapjq this shouldn't be needed if the container runtime does it for you (e.g. with `nvidia-docker` or a device plugin).

Reply:
@flx42 that means I have to install nvidia-docker in prior if we want to use this functionality? That we may have to note in the ReadMe as well IIUC.

Reply:
I tested this by directly running it using:
```
sudo ./cadvisor -alsologtostderr
```
Kubelet also runs directly on the host, so it would work there as well.

You are right, we need to document that to run inside a container and still get GPU metrics, you will need to either use `nvidia-docker` or mount the libraries and setup some environment variables like what device-plugins do.